api/src/routing.js

```diff
@@ -700,6 +760,7 @@ app.all(appPrefix + '*', authorization.setAuthorized);
 // block offline users requests from accessing CouchDB directly, via Proxy
 // requests which are authorized (fe: by BulkDocsHandler or DbDocHandler) can pass through
 // unauthenticated requests will be redirected to login or given a meaningful error
+app.use(authorization.handleAuthentication(true));
```

Comment:
As this is the only place where we pass the param I think it makes more sense to have two different functions, neither of which take a parameter. By using a descriptive function name it'll be clearer what it does, rather than having to go into the function to work out what `true` means.

Reply:
Yea, that was on my irk list as well :) Changed.